src/diffeqs/laplace.jl

```diff
@@ -0,0 +1,296 @@
+import DomainSets.ClosedInterval
+
+# from https://tutorial.math.lamar.edu/Classes/DE/Laplace_Table.aspx
+transform_rules(f, t, F, s) = Symbolics.Chain([
```

Comment:
I wonder if `SymbolicUtils.@cache` will play nicely with this. It could save a lot of time considering that this is called recursively. This also applies to the previous comments about `@rule` in recursive functions.